src/mo_def/arrange.ml

```diff
@@ -98,7 +98,8 @@ module Make (Cfg : Config) = struct
         Atom (if sugar then "" else "=");
         exp e'
       ]
-    | CallE (e1, ts, e2)  -> "CallE"   $$ [exp e1] @ inst ts @ [exp e2]
+    | CallE (None, e1, ts, e2)  -> "CallE"      $$ [exp e1] @ inst ts @ [exp e2]
+    | CallE (Some par, e1, ts, e2) -> "CallE()" $$ [exp par] @ [exp e1] @ inst ts @ [exp e2]
```

Comment:
It turns out that vscode actually uses arrange.ml to communicate type info with the plugin (not pretty, but true). This means that changing the output format might affect vscode.

To avoid, this it might be better to follow and adapt the current logic for migration expressions (see `include_migration` and `migration eo`. Maybe it's enough to rename these to `include_parenthetical` `parethetical eo` and use in CallE and AsyncE cases

Reply:
The idea was to drop the parenthetical completely when outputting syntax for vscode.

Reply:
Of course the alternative is to check the plug in still works and adapt if necessary....

Reply:
I went the `include_parenthetical` route!